internal/datastore/datastore.go

```diff
@@ -39,14 +39,15 @@ type Datastore interface {
 	RemoveFlairTemplate(ctx context.Context, flairTemplate model.FlairTemplate) (*model.FlairTemplate, error)
 	GetTotalParticipantCountByDiscussionID(ctx context.Context, discussionID string) int
 	GetParticipantByID(ctx context.Context, participantID string) (*model.Participant, error)
+	GetParticipantsByIDs(ctx context.Context, ids []string) ([]*model.Participant, error)
```

Comment:
do we want to return a map or a slice of pointers? GetDiscussionsByIDs returns a map but the map isn't really used

Reply:
Maps are almost always superior here. Slices are weird if things aren't found or throw errors. A map gives you a lot more leeway.

That said it's not a major deal either way.